Src/AutoFakeItEasy2.UnitTest/AutoFakeItEasy2.UnitTest.csproj

```diff
@@ -62,7 +62,7 @@
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\AutoFakeItEasy2\AutoFakeItEasy2.csproj">
-      <Project>{b6b79e32-ee90-4ae1-bd32-f50b5a0c2b40}</Project>
+      <Project>{68A3F347-7D9D-4020-A3D5-F127234DD429}</Project>
```

Comment:
It appeared that AutoFakeItEasy2 projects were not created correctly - they were copy-pasted from the original AutoFakeItEasy ones.

After I added all projects to the single solution, the conflicts appeared. Therefore, you see 3 modifications - 2 project IDs and single project reference were updated.